ui/js/component/app/view.jsx

```diff
@@ -21,6 +22,8 @@ class App extends React.PureComponent {
       checkUpgradeAvailable();
     }
 
+    initCheckUpgradeTimer();
```

Comment:
doesn't this check for an upgrade in the if statement above? maybe replace checkUpgradeAvailable on line 22 with this line?

Reply:
The timer doesn't trigger a check for an upgrade immediately. That's why I kept the ``checkUpgradeAvailable();``, so the app does check for an upgrade when it starts.

Reply:
If this can (or should) wait until the daemon is ready, we have other actions that are first-run actions here:

https://github.com/lbryio/lbry-app/blob/master/ui/js/redux/actions/app.js#L177

If this should happen before the daemon is ready, then this is an appropriate place.